templates/app-template-lit-element-typescript/src/index.ts

```diff
@@ -0,0 +1,37 @@
+import { customElement, LitElement, property, html, css } from "lit-element";
+import { classMap } from "lit-html/directives/class-map";
```

Comment:
This is the import that can't be resolved with Snowpack

Reply:
We should improve this error message. That file doesn't exist since it's missing a file extension. It should be:`lit-html/directives/class-map.js`

Reply:
Aaaaah, I see, ok then!